src/sql/workbench/contrib/notebook/browser/notebookLinkHandler.ts

```diff
@@ -0,0 +1,131 @@
+import { URI } from 'vs/base/common/uri';
```

Comment:
copyright header

(did it not warn you about missing this when committing? That should be done in the precommit hooks I thought)

Reply:
Never got a warning about it. Will add it though.